tests/nlu/test_components.py

```diff
@@ -172,3 +172,17 @@ def test_can_handle_language_guard_clause(
     with pytest.raises(RasaException) as excinfo:
         SampleComponent.can_handle_language("random_string")
     assert expected_exec_msg in str(excinfo.value)
+
+
+async def test_validate_requirements_raises_exception_on_component_without_name(
+    tmp_path,
```

Comment:
Missing the type for `tmp_path` here.

Reply:
good point, added them to all tests in this file while I was at it 🙂